packages/trie/test/trie/secure.spec.ts

```diff
@@ -6,8 +6,8 @@ import {
   hexToBytes,
   utf8ToBytes,
 } from '@ethereumjs/util'
-import { createHash } from 'crypto'
 import { keccak256 } from 'ethereum-cryptography/keccak.js'
+import { sha256 } from 'ethereum-cryptography/sha256.js'
```

Comment:
Fixes error: 
`Error: Module "crypto" has been externalized for browser compatibility. Cannot access "crypto.createHash" in client code.  See https://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility for more details.`

Reply:
for some things, the `crypto` polyfill works, but `crypto.createHash` is not available.  
the `ethereum-cryptography` packages do work in the browser, though.